programs/stake_tests/tests/stake_instruction.rs

```diff
@@ -80,11 +87,105 @@ fn test_stake_account_delegate() {
     let account = bank.get_account(&staker_pubkey).expect("account not found");
     let stake_state = account.state().expect("couldn't unpack account data");
     if let StakeState::Stake(stake) = stake_state {
-        assert_eq!(stake.stake, 200);
+        assert_eq!(stake.stake, 20000);
     } else {
         assert!(false, "wrong account type found")
     }
 
+    // Reward redemption
+    // Submit enough votes to generate rewards
+    let mut old_epoch = bank.epoch();
+    while bank.epoch() != old_epoch + 1 {
+        bank = Arc::new(Bank::new_from_parent(
+            &bank,
+            &Pubkey::default(),
+            1 + bank.slot(),
+        ));
+
+        let bank_client = BankClient::new_shared(&bank);
+
+        let message = Message::new_with_payer(
+            vec![vote_instruction::vote(
+                &vote_pubkey,
+                &vote_pubkey,
+                vec![Vote::new(
+                    (bank.slot() - 1) as u64,
```

Comment:
could use bank.parent().slot(), or take slot and hash at the top (after calling bank.freeze() on the parent)